rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -4115,7 +4115,7 @@ fn cycles_correct_if_upgrade_succeeds() {
 #[test]
 fn cycles_correct_if_upgrade_fails_at_validation() {
     let mut test = ExecutionTestBuilder::new()
-        .with_allocatable_compute_capacity_in_percent(50)
+        .with_rate_limiting_of_instructions()
```

Comment:
Could we do this in the other test, too? (to avoid the need to install two times)

Reply:
Done.